src/main/frontend/pipeline-console-view/pipeline-console/main/StageView.tsx

```diff
@@ -196,14 +196,23 @@ export default class StageView extends React.Component {
   render() {
     return (
       <React.Fragment>
+        {/*<div*/}
+        {/*  key={`stage-summary-${*/}
+        {/*    this.props.stage ? this.props.stage.id : "unk"*/}
+        {/*  }`}*/}
+        {/*>*/}
+        {/*  {this.renderStageDetails()}*/}
```

Comment:
would be good to re-add a version of this.
It doesn't have to take up as much space and maybe is collapsed by default but stage time and being able to view a whole stages logs is very valuable